packages/app-project/pages/_app.js

```diff
@@ -38,7 +38,7 @@ export default class MyApp extends App {
 
     if (pageProps.isServer) {
       // cookie is in the next.js context req object
-      const mode = getCookie(context, 'mode') || ''
+      const mode = getCookie(context, 'mode') || 'light'
```

Comment:
This is essentially setting a default in two places. Instead, we could use:

```
const mode = getCookie(context, 'mode') || undefined
```

`getCookie(context, 'mode')` will return an empty string if not set, which will break the model; `undefined` won't.

Reply:
That's fixed now.